src/aks-preview/setup.py

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env python
+
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+from codecs import open as open1
+from setuptools import setup, find_packages
+try:
+    from azure_bdist_wheel import cmdclass
+except ImportError:
+    from distutils import log as logger
+    logger.warn("Wheel is not available, disabling bdist_wheel hook")
+    cmdclass = {}
+
+VERSION = "0.0.1"
+CLASSIFIERS = [
+    'Development Status :: 5 - Production/Stable',
+    'Intended Audience :: Developers',
+    'Intended Audience :: System Administrators',
+    'Programming Language :: Python',
+    'Programming Language :: Python :: 2',
+    'Programming Language :: Python :: 2.7',
+    'Programming Language :: Python :: 3',
+    'Programming Language :: Python :: 3.4',
+    'Programming Language :: Python :: 3.5',
+    'Programming Language :: Python :: 3.6',
+    'License :: OSI Approved :: MIT License',
+]
+
+DEPENDENCIES = []
+
+with open1('README.md', 'r', encoding='utf-8') as f:
+    README = f.read()
+with open1('HISTORY.md', 'r', encoding='utf-8') as f:
+    HISTORY = f.read()
+
+setup(
+    name='aks-preview',
```

Comment:
I just want to be clear, if this is the name for the extension, future preview features for aks will also need to be bundled with this extension.

Reply:
Thanks for your advice. We are ok with this name.